happex/main.py

```diff
@@ -58,6 +100,6 @@ def get_generosity():
 
 
 @app.route("/country/<country>")
-def get_country(country):
-    paffects = app.db.get(country_name=country, cols=("year", "paffect"))
+def get_country(ctry):
+    paffects = app.db.get(country_name=ctry, cols=("year", "paffect"))
```

Comment:
Calling this argument `country` was possibly causing a pylint warning due to the global country I mentioned in an earlier comment, but not changing the name in all of the places it is used causes an exception when the route is run. Either `ctry` or `country` is fine, but make sure you test the changes.

Reply:
I did change everything here to work as needed; the issue stems from what you originally put in there - there's no get_country function in the database unless you have it on your end from where Michael or Sumitra put it in.